### PR TITLE
PAPRIKA BROKE HEADSETS

### PR DESCRIPTION
**Pull request by Niknakflak**
The icon state for security headsets was changed to sec_headset_old.  After reviewing all the code, there was no reason for the icon state to be changed, so I changed it back to sec_headset and it fixed everything.

### CODE REVIEW

**Comment by Niknakflak** [comment]
I hope I didn't mess anything up, it's been awhile since I used github.

**Comment by ghost** [comment]
paprika the king of sprite edits

also it's because i was going to replace sec headsets with bowman headsets but cheridan had a better idea and that's simply making them alternatives, which is what i did.

**Comment by MrSnapwalk** [comment]
You committed the .int file, which is a bad thing due to compile lag and stuff.

**Comment by Niknakflak** [comment]
I figured that out now, how do I get rid of it

File: NTstation13.int [hunk 1]
```diff
@@ -1,6 +1,11 @@
 // BEGIN_INTERNALS
 /*
 MAP_ICON_TYPE: 0
+LAST_COMPILE_VERSION: 503.1224
+WINDOW: icons\obj\radio.dmi;code\game\objects\items\devices\radio\headset.dm
+DIR: code code\game code\game\machinery code\game\machinery\computer code\game\objects code\game\objects\items code\game\objects\items\devices code\game\objects\items\devices\radio code\game\objects\structures code\game\verbs code\modules icons icons\obj 
+FILE: icons\obj\radio.dmi
+LAST_COMPILE_TIME: 1404101347
```
Comment (ghost) [inline NTstation13.int]: just remove these 5 lines from your .int file on the branch and commit it

**Comment by ike709** [comment]
All this for one icon state.

**Comment by ghost** [comment]
:+1:

**Comment by MrSnapwalk** [comment]
WELCOME TO GIT, ENJOY YOUR STAY

File: NTstation13.int [hunk 2]
```diff
@@ -1,6 +0,0 @@
-// BEGIN_INTERNALS
-/*
-MAP_ICON_TYPE: 0
-AUTO_FILE_DIR: OFF
-*/
-// END_INTERNALS
```
Comment (ghost) [inline NTstation13.int:6]: YOU DELETED TOO MUCH READD THIS

**Comment by ghost** [comment]
Merge this posthaste.

**Comment by ike709** [comment]
Four commits to change one icon state. GG.

**Comment by Niknakflak** [comment]
Look how good of a coder I am guys

**Comment by ike709** [comment]
You should be head coder

**Comment by ghost** [comment]
valve merge this please

**Pull request merged**